### PR TITLE
product model setup

### PR DESCRIPTION
**Pull request by thekontinet**

### CODE REVIEW

**Comment by Ovie33** [comment]
i am a child of god

**Comment by FreemannFirms** [comment]
Why are you too good?@Thekontinet

**Comment by Ovie33** [comment]
how are you guys

File: migrations/20220725160818-create-product.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      title: {
+        type: Sequelize.STRING
+      },
+      price: {
+        type: Sequelize.INTEGER
+      },
+      quantity: {
+        type: Sequelize.INTEGER
+      },
+      userId: {
```
Comment (thekontinet) [inline migrations/20220725160818-create-product.js:20]: Please use underscore notation

**Comment by Ovie33** [comment]
favour stop sleeping

**Comment by FreemannFirms** [comment]
We are fine..what about you?

File: migrations/20220725160818-create-product.js (hunk 1 above)
Comment (Ovie33) [reply migrations/20220725160818-create-product.js:20]: hello

**Comment by Ovie33** [comment]
> We are fine..what about you?

i am doing well i have diabeties

**Pull request merged**